pyquil/operator_estimation.py

```diff
@@ -824,10 +825,13 @@ def measure_observables(qc: QuantumComputer, tomo_experiment: TomographyExperime
             # 2.3 Gather together all the symmetrized results
             bitstrings = reduce(lambda d1, d2: _stack_dicts(d1, d2), list_bitstrings_symm)
 
-        else:
+        elif readout_symmetrize is None:
             # 2. Run the experiment
             bitstrings = qc.run_and_measure(total_prog, n_shots)
 
+        else:
+            raise ValueError("'readout_symmetrize' must be either 'exhaustive' or None")
```

Comment:
It is uncommon IME to use quotes around a variable name in an error message

Reply:
Updating.